### DISCUSSION TITLE
Error!!! en la corrección automática 04-JS-III

### DISCUSSION THREAD

**Discussion by doorsjaime** [atralice/Curso.Prep.Henry#99] [General]
Buenas tengo un problema con el ultimo ejercicio de 04-JS-III 

function continueStatement(numero) {
  //Iterar en un bucle aumentando en 2 el numero recibido hasta un límite de 10 veces.
  //Guardar cada nuevo valor en un array.    
  //Devolver el array
  //Cuando el número de iteraciones alcance el valor 5, no se suma en ese caso y se continua con la siguiente iteración
  //Pista: usa el statement 'continue'
  // Tu código:
i = 0;
array = [ ];
suma = numero;
while (i < 10) {
  i++;
  suma = suma + 2;
  if (i == 5) {continue;} 
  array.push (suma)
  }
  return array
}

// ejemplos de mi resolución
continueStatement(50)
(9) [52, 54, 56, 58, 62, 64, 66, 68, 70]
continueStatement(-4)
(9) [-2, 0, 2, 4, 8, 10, 12, 14, 16]

//pero la corrección automática sale lo siguiente
expect(continueStatement(50)).toEqual([52, 54, 56, 58, 60, 62, 64, 66, 68]);
    expect(continueStatement(-4)).toEqual([-2, 0, 2, 4, 6, 8, 10, 12, 14]);
    
lo cual en la iteración numero 5 donde debería saltar a la siguiente no lo hace. 

Corríjanme si estoy equivocado

**Comment by jmmontalvog** [comment]
Yo también no le entendía al principio, pero cuando dice que la iteración numero 5 no debe realizarse es literal, te saltas esa iteración entonces, de las 10 que ibas a realizar solo harás 9, pero eso no significa que tengas que saltarte la suma que tedio en la iteración 5.
Entonces lo que llevabas de suma hasta la iteración 4 (58), cuando llegues a la iteración 6 debes continuar con la suma y este debe de ser 60.
Te recomiendo que utilices un for y otro contador.

**Comment by Hosmmer** [comment]
function comprobarBreak(x,nuintera) {
  var i = x;
   var j = 0;
  var suma = 0;
  var array = [];
   while (i < 10) {
    
 	var suma=i+=2;   
     array.push(j);
     //array.push(suma);
     
      if(j==nuintera){
        array.push('Se interrumpió la ejecución');
     	break;
      }
  	j++; 
   }
   return array;
}

console.log(comprobarBreak(0,4));

**Comment by fgarciamatar** [comment]
var array = [];
   var suma = num;
    for (var i = 0; i < 10 ; i++){
       if ( i+1 === 5 ){
         continue;
        } else {
           suma = suma + 2;
         array.push(suma);
        } }
  return array;
